src/components/HeaderFrame.tsx

```diff
@@ -42,7 +44,7 @@ function HeaderFrame() {
         profileData={profileData}
         isSticky={pathname === "/shared" ? true : false}
```

Comment:
- 이렇게 작성해도 동일합니다.
  ```suggestion
          isSticky={pathname === "/shared"}
  ```
- 그리고 sticky한 페이지가 지금 당장은 하나일지라도 추후에 추가될 수도 있기 때문에, 해당 페이지들에 대한 경로를 배열로 관리하면 더 좋겠네요!